src/components/Form.tsx

```diff
@@ -71,6 +71,9 @@ export class UnwrappedForm extends Component<IFormWrappedProps> {
     return fillInFieldSets(this.props.fieldSets);
   }
 
+  private hasErrors = (fieldsError: any) =>
```

Comment:
As much as I like this syntax, I think the standard of the project is to use `{}` and a return inside. @rkuykendall has the last word on this.

Reply:
I don't dislike this syntax tt all, but yeah, I think that we should keep things consistent with the rest of the project and expand this.